src/main/java/uk/ac/ebi/ampt2d/registry/entities/Property.java

```diff
@@ -97,11 +98,23 @@
 
     @CreatedDate
     @Column(updatable = false)
+    @Convert(converter = ZonedDateAttributeConverter.class)
     private ZonedDateTime createdDate;
 
     @LastModifiedDate
+    @Convert(converter = ZonedDateAttributeConverter.class)
     private ZonedDateTime lastModifiedDate;
 
+    public Property() {
+    }
+
+    public Property(String id, Type type, Meaning meaning, String description) {
+        this.id = id;
+        this.type = type;
+        this.meaning = meaning;
```

Comment:
I now wonder what `meaning` is and what is it different from `description`. Are they essentially the same thing?

Reply:
I think it should be description but , meaning mostly they use it for portal side in broad.

Reply:
I think we need to clarify on this. If it is redudant we should make changes.

Reply:
+1 to @Zhicheng-Liu comment about clarification. One of the goals of the new implementation was to clean up the most questionable aspects of the existing API.

Reply:
Broad has asked to keep the column for now. in future they would try to remove it.